features/upgrade/storage/upgrade.feature

```diff
@@ -504,6 +511,7 @@ Feature: Storage upgrade tests
   @upgrade
   @network-ovnkubernetes @network-openshiftsdn
   @hypershift-hosted
+  @inactive
```

Comment:
Instead of inactive, perhaps we should remove it as the test case have been removed in Polarion.

Reply:
It makes sense, updated.